code/modules/clothing/suits/arfsuits_unused.dm

```diff
@@ -312,35 +377,57 @@ Suits. 0-10 in its primary value, slowdown 0, various utility
 	icon_state = "ncr_codressjack"
 	item_state = "ncr_codressjack"
 
-
 //NCR Ranger
 /obj/item/clothing/suit/toggle/armor/rangerrecon
 	name = "ranger recon duster"
 	desc = "A thicker than average duster worn by NCR recon rangers out in the field. It's not heavily armored by any means, but is easy to move around in and provides excellent protection from the harsh desert environment."
 	icon_state = "duster_recon"
 	item_state = "duster_recon"
 	salvage_loot = list(/obj/item/stack/crafting/armor_plate = 4)
+	armor = ARMOR_VALUE_LIGHT
+	stiffness = LIGHT_STIFFNESS
+	armor_tier_desc = ARMOR_CLOTHING_LIGHT
+	slowdown = ARMOR_SLOWDOWN_LIGHT * ARMOR_SLOWDOWN_LESS_T1 * ARMOR_SLOWDOWN_GLOBAL_MULT
+	armor_tokens = list(ARMOR_MODIFIER_UP_DT_T2, ARMOR_MODIFIER_UP_BULLET_T1)
 
 /obj/item/clothing/suit/armor/trailranger
 	name = "ranger vest"
 	desc = "A quaint little jacket and scarf worn by NCR trail rangers."
 	icon_state = "cowboyrang"
 	item_state = "cowboyrang"
 	salvage_loot = list(/obj/item/stack/crafting/armor_plate = 4)
+	armor = ARMOR_VALUE_LIGHT
+	stiffness = LIGHT_STIFFNESS
+	armor_tier_desc = ARMOR_CLOTHING_LIGHT
+	slowdown = ARMOR_SLOWDOWN_LIGHT * ARMOR_SLOWDOWN_GLOBAL_MULT
+	armor_tokens = list(ARMOR_MODIFIER_UP_DT_T2, ARMOR_MODIFIER_UP_BULLET_T1)
 
-/obj/item/clothing/suit/armor/modif_r_vest
+/obj/item/clothing/suit/armor/trailranger/modif_r_vest
 	name = "subdued ranger vest"
 	desc = "A quaint little jacket and scarf worn by NCR trail rangers. This one has the leather bleached and the scarf dyed black."
 	icon_state = "modif_r_vest"
 	item_state = "modif_r_vest"
 	salvage_loot = list(/obj/item/stack/crafting/armor_plate = 4)
 
+/obj/item/clothing/suit/armor/medium/combat/desert_ranger/patrol
```

Comment:
Good move putting this with the rest of the armors, guess the furries really liked Ranger patrol armor.